notifications/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from users.models import Member
+from django.utils import timezone
+from datetime import timedelta
+
+# TODO:
+# Consider merging notifications of the same type. How do we do this???
+# COULD MERGE BY URL
+
+# Create your models here.
+class Notification(models.Model):
+	member = models.ForeignKey(Member, related_name='notifications_owned', on_delete=models.CASCADE)
+	notify_type = models.CharField(max_length=32)
```

Comment:
Nitpick: tempted to suggest `notification_type` or just `type` if possible

Reply:
`type` is reserved by Python. How about `notif_type`?

Reply:
Not a huge fan of shorthand, but I do like `notif_type` more than `notify_type` :+1:

Reply:
Applied. :accept: